### DISCUSSION TITLE
I got a notification that this doesn't work with Foundry 9.x

### DISCUSSION THREAD

**Discussion by MortalxWombat** [zarmstrong/foundry-pathbuilder2e-import#15] [General]
I just bought the paid version of pathbuilder after I stumbled on this, but unfortunately it says it's not working at the moment.  Is there an ETA on a fix?  Thank you so much for your time and work on this!

**Comment by zarmstrong** [comment]
Please see https://github.com/zarmstrong/foundry-pathbuilder2e-import/issues/11#issuecomment-1025448377
